custom_components/openhasp/__init__.py

```diff
@@ -554,7 +556,7 @@ async def async_change_page(self, page):
         if self._statusupdate:
             num_pages = self._statusupdate[HASP_NUM_PAGES]
 
-            if page <= 0 or page > num_pages:
+            if page <= 0 or int(page) > int(num_pages):
```

Comment:
This is unrelated to this PR, if there is an issue please open a separate PR

Reply:
I intended to, but the change got included anyway... unfortunate way of working in the same branch.. you can cherry pick it out and I will load it into another once this one is merged.

There is an issue that is fixed with this change, the display stops or refuses to dynamically update and throws an error in the logs because (most likely) num_pages contains a string, while page contains an int, resulting in a python type error.

Reply:
I can't cherry pick on the PR... please make a commit to revert this change

Reply:
Ok, will do.